app/views/devise/registrations/new.html.erb

```diff
@@ -38,59 +37,80 @@
     <div class="card-body">  
       <h5 class="card-title">User</h5>
         <div>
-             <%= f.label :picture %>
-             <%= f.text_field :picture, autofocus: true %>
+           <%= f.label :picture %>
+           <%= f.text_field :picture, autofocus: true %>
         </div>
         <div>
-             <%= f.label :first_name %>
-             <%= f.text_field :first_name, autofocus: true %>
-        
-             <%= f.label :last_Name %>
-             <%= f.text_field :last_name, autofocus: true %>
+           <%= f.label :first_name %>
+           <%= f.text_field :first_name, autofocus: true %>
+      
+           <%= f.label :last_name %>
+           <%= f.text_field :last_name, autofocus: true %>
         </div>
         <div>
-             <%= f.label :Gender %>
-             <%= f.select :gender,[ 'Female', 'Male', 'Other', 'N/A' ], :prompt => 'Select One', autofocus: true %>
+          <%= f.label :gender %>
+          <%= f.select :gender,[ 'Female', 'Male', 'Other', 'N/A' ], :prompt => 'Select One', autofocus: true %>
         </div>   
         <div>
-             <%= f.label :Age %>
-             <%= f.select :age, options_for_select(18..100), :prompt => "Age" ,autofocus: true %>
+          <%= f.label :age %>
+          <%= f.select :age, options_for_select(18..100), :prompt => "Age" ,autofocus: true %>
         </div>
         <div>
-             <%= f.label :Address %>
-             <%= f.text_field :street_address, autofocus: true %>
+          <%= f.label :address %>
```

Comment:
The parameter name that you pass to f.label needs to exactly match the parameter name that you pass to f.text_field. In this case, that is :street_address. If you want the label to show different text, you can do that easily by passing in an additional parameter to f.label:

```suggestion
          <%= f.label :street_address, "Address" %>
```